### DISCUSSION TITLE
How do I access the code from the demos page?

### DISCUSSION THREAD

**Discussion by Bijig0** [Comcast/react-data-grid#3419] [Q&A] [answered]
I want to copy one of the demos, where is the source code for them?

**Comment by thomassmartinez** [comment]
here: https://github.com/adazzle/react-data-grid/tree/main/website/demos

**Reply by bobwhitleywork** [reply]
This URL is no longer any good. Is there an updated one we can use?

**Comment by Ben-CA** [comment, marked as answer]
As as far as documentation, I agree it could be greatly expanded. There's the Readme:
https://github.com/adazzle/react-data-grid/blob/main/README.md

The examples:
https://adazzle.github.io/react-data-grid/#/CommonFeatures

And the code behind the examples:
https://github.com/adazzle/react-data-grid/blob/main/website/routes/AllFeatures.lazy.tsx

**Reply by amanmahajan7** [reply]
https://github.com/adazzle/react-data-grid/tree/main/website/routes
